<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <link rel="stylesheet" href="style/base.css" type="text/css" />


<!-- // modif : lecorguille -->
    <script src="W4M/jquery-ui-1.11.4.custom/external/jquery/jquery.js"></script>
    <script src="W4M/jquery-ui-1.11.4.custom/jquery-ui.js"></script>
    <link rel="stylesheet" href="W4M/jquery-ui-1.11.4.custom/jquery-ui.css">

    <script>
        $(document).ready(function () {
            $(function() {
                $( "#W4M #accordion > div" ).accordion({ heightStyle: "content",collapsible: true });}
            );
            $(function() {
                $( "#W4M #changelog_accordion" ).accordion({ heightStyle: "content",collapsible: true });
            })
            $(function() {
                $( "#W4M #tutorial_accordion" ).accordion({ heightStyle: "content",collapsible: true });
            })
            /**$("#W4M #tutorial_accordion > div").accordion({ heightStyle: "content",collapsible: true });*/
            $(function() {
                $( "#W4M .collapseclose" ).accordion({ active: false });}
            );
        });
        // fonction toogle for diplay block <-> none
        function toogleDisplay(id)
        {
        if (document.getElementById(id).style.display == "block") {
            document.getElementById(id).style.display = "none";
        }
        else {
            document.getElementById(id).style.display = "block";
        }
        }

    </script>

    <style>

        /*FROM base.css*/
        .errormessagelarge,.warningmessagelarge,.donemessagelarge,.infomessagelarge{padding:15px;margin-bottom:17px;border:1px solid transparent;border-radius:3px;min-height:36px;padding-left:52px;background-image:url(error_large.png);background-repeat:no-repeat;background-position:10px 8px}.errormessagelarge h4,.warningmessagelarge h4,.donemessagelarge h4,.infomessagelarge h4{margin-top:0;color:inherit}
        .errormessagelarge .alert-link,.warningmessagelarge .alert-link,.donemessagelarge .alert-link,.infomessagelarge .alert-link{font-weight:bold}
        .errormessagelarge>p,.warningmessagelarge>p,.donemessagelarge>p,.infomessagelarge>p,.errormessagelarge>ul,.warningmessagelarge>ul,.donemessagelarge>ul,.infomessagelarge>ul{margin-bottom:0}
        .errormessagelarge>p+p,.warningmessagelarge>p+p,.donemessagelarge>p+p,.infomessagelarge>p+p{margin-top:5px}
        .errormessagelarge{background-color:#f9c7c5;border-color:#dd1c15;color:#7e2a27;padding-left:52px}.errormessagelarge hr{border-top-color:#c61913}
        .errormessagelarge .alert-link{color:#571d1b}
        .warningmessagelarge{background-color:#fce1ba;border-color:#e28709;color:#80571e;padding-left:52px;background-image:url(warn_large.png)}.warningmessagelarge hr{border-top-color:#c97908}
        .warningmessagelarge .alert-link{color:#573b14}
        .donemessagelarge{background-color:#aff1af;border-color:#20b420;color:#295f29;padding-left:52px;background-image:url(ok_large.png)}.donemessagelarge hr{border-top-color:#1c9e1c}
        .donemessagelarge .alert-link{color:#193b19}
        .infomessagelarge{background-color:#a6e4f7;border-color:#1197c0;color:#1f5566;background-image:url(info_large.png)}.infomessagelarge hr{border-top-color:#0f85a8}
        .infomessagelarge .alert-link{color:#13343f}

        #W4M  ul {
            list-style-type: none;
            margin-left:0;
        }
        #W4M  ul > li > ul {
            list-style-type: disc;
            margin-left:20px;
        }
        #W4M  ul > li > ul > li > ul {
            list-style-type: square;
            margin-left:20px;
        }

        video {
            padding-top: 15px;
            padding-bottom: 25px;
        }
        .donemessagelarge, .warningmessagelarge, .infomessagelarge {
            padding-top:2px;
            padding-bottom:5px;
        }
        h1, h2 {
            margin-top:8.5px;
        }


    </style>
<!-- // modif fin : lecorguille -->

</head>
<body>
    <div class="document">
        <div width="auto" align="center">
                <img src="W4M/logo-ifb-mono-metabohub_2.1.png" style="max-width:1200px;width:100%;" />
        </div>

        <div class="donemessagelarge">
            <h1>Current version :  3.2 - 19/09/2018</h1>
            <strong>Publication:</strong> Franck Giacomoni, Gildas Le Corguillé, Misharl Monsoor, Marion Landi, Pierre Pericard, Mélanie Pétéra, Christophe Duperier, Marie Tremblay-Franco, Jean-François Martin, Daniel Jacob, Sophie Goulitquer, Etienne A. Thévenot and Christophe Caron (2014). <strong>Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics</strong>. Bioinformatics  <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/12/18/bioinformatics.btu813" target="_blank">doi:10.1093/bioinformatics/btu813</a><br/><br/>
            <b>Help and support: <a href="mailto:support@workflow4metabolomics.org">support@workflow4metabolomics.org</a></b>
        </div>

        <div class="infomessagelarge">
            <h2>Latest news</h2>
            <ul>
                <li>UPGRADE/NEW - LCMS/Preprocessing - xcms.* (3.0.0): upgrade the xcms version from 1.46.0 to 3.0.0. Note that there aren't any retro compatibility between xcms1.46.0 and xcms3.0.0. You have to run the whole workflow from scratch.</li>
                <li>NEW in Docker - ALL/Statistics - mixmodel ANOVA for repeated measures statistics</li>
                <li>NEW in Docker - ALL/Statistics - ASCA Splitting of the total variance into independent blocks according to the experimental factors and multivariate analysis (SCA) of each block</li>
                <li>NEW in Docker - LCMS/Annotation - bank_inhouse search by accurate mass (and by Retention time) on a local bank</li>
                <li>NEW in Docker - NMR/Annotation - NMR_Annotation Annotation of complex mixture NMR spectra and metabolite proportion estimation</li>
                <li>UPGRADE
                    <ul>
                        <li>LCMS/Preprocessing - camera.*</li>
                        <li>FIAMS/Preprocessing - Profia</li>
                        <li>GCMS/Preprocessing - MetaMS</li>
                        <li>NMR/Preprocessing - NMR preprocessing, NMR alignment, NMR bucketing</li>
                        <li>ALL/Quality Metric - Normalization, Transformation, Quality Metrics, Multilevel, </li>
                        <li>ALL/Statistics - Univariate, Multivariate, Anova, </li>
                        <li>LCMS/Annotation - HMDB MS search, Lipidmaps</li>
                    </ul>
                </li>
            </ul>
        </div>

        <div id="W4M">
            <div id="accordion">
                <div> <!-- class="collapseclose">-->
                    <h3>Tutorials</h3>
                    <div  id="tutorial_accordion">
                        <h4>HOW TO</h4>
                        <div>
                            <table style="width: 100%;" border="1" cellspacing="1" cellpadding="1"><thead><tr><th>CATEGORY</th><th>FIELD</th><th>Description</th><th>PDF</th></tr></thead><tbody><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Prepare input datasets</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/aaeb31.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Import datasets</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/8880e7.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Cleanup datasets</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/f0fd7c.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Build And Configure A Workflow</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/9bc79d.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Galaxy</td><td style="width: 150px;">&nbsp;</td><td style="width: 369px;">Share Histories And Workflows</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/8e8802.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Preprocessing</td><td style="width: 150px;">LC-MS</td><td style="width: 369px;">Perform Preprocessing using XCMS</td><td style="width: 100px;"><a href="http://abims.sb-roscoff.fr/mypydio/data/public/084ad1.php?dl=true">Download</a></td></tr><tr><td style="width: 150px;">Quality Control</td><td style="width: 150px;">Common</td><td style="width: 369px;">Format Data For Postprocessing</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToFormatDataForPostprocessing_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Batch correction</td><td style="width: 150px;">LC-MS and GC-MS</td><td style="width: 369px;">Perform Drift And Batch Correction</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformDriftAndBatchCorrection_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Statistics</td><td style="width: 150px;">Common</td><td style="width: 369px;">Perform Univariate Analyzes</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformUnivariateAnalyzes_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Statistics</td><td style="width: 150px;">Common</td><td style="width: 369px;">Perform Multivariate Analyzes</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformMultivariateAnalyzes_v02.pdf">Download</a></td></tr><tr><td style="width: 150px;">Annotation</td><td style="width: 150px;">LC-MS</td><td style="width: 369px;">Perform LCMS Annotations</td><td style="width: 100px;"><a href="http://web11.sb-roscoff.fr/download/w4m/howto/w4m_HowToPerformAnnotations_v05.pdf">Download</a></td></tr><tr><td style="width: 150px;">Annotation</td><td style="width: 150px;">GC-MS</td><td style="width: 369px;">Use NIST</td><td style="width: 100px;"><a href="http://application.sb-roscoff.fr/download/w4m/howto/w4m_HowToUseNIST_V01.pdf">Download</a></td></tr></tbody></table>

                            <br/><hr/>
                            <h5 id="single_files">Use single files as input and run xcmsSet in parallel</h5>
                            <video style="align:center;max-width:1280px;width:95%;" controls="controls" scale="tofit" >
                                <source src="http://download.workflow4metabolomics.org/docs/170510_galaxy_xcms_dataset_collection.m4v" type="video/mp4">
                            </video>
                            <a href="http://download.workflow4metabolomics.org/docs/170510_galaxy_xcms_dataset_collection.m4v">Download the video</a>
                        </div>
                        <h4>W4M courses 2015</h4>
                        <div>
                                <table style="width: 100%;" border="1" cellspacing="1" cellpadding="1"><thead><tr><th scope="col">W4M courses 2015</th><th scope="col">Author(s)</th><th scope="col">PDF</th></tr></thead><tbody><tr><td>Course presentation</td><td>Christophe Caron, Franck Giacomoni</td><td><a href="/sites/workflow4metabolomics.org/files/files/00_course_presentation_fg-cc_20150918.pdf" target="_blank">Download</a></td></tr><tr><td>Initiation</td><td>Gildas Le Corguillé, Misharl Monsoor, Marion Landi</td><td><a href="/sites/workflow4metabolomics.org/files/files/galaxy-init-1.05_printable.pdf" target="_blank">Download</a></td></tr><tr><td>LC-MS -&nbsp;Pre-processing (xcms)</td><td>W4M Core Team</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_xcms_CAMERA_LC-MS.pdf" target="_blank">Download</a></td></tr><tr><td>MS data processing - Filtering and correcting data</td><td>W4M Core Team</td><td><a href="/sites/workflow4metabolomics.org/files/files/MS_data_processing.pdf" target="_blank">Download</a></td></tr><tr><td>Using&nbsp;Galaxy4Metabolomics -&nbsp;W4M table format for&nbsp;Galaxy</td><td>W4M Core Team</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_TableFormatForGalaxy_150908.pdf" target="_blank">Download</a></td></tr><tr><td>How to check the format&nbsp;of the 3 .tsv tables</td><td>Etienne Thévenot</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_Appendix_CheckFormat_150923.pdf" target="_blank">Download</a></td></tr><tr><td>How to perform statistical analysis ? - Introduction</td><td>Etienne Thévenot, Marie Tremblay-Franco, Mélanie Pétéra, Gildas Le Corguillé</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_1_Introduction_150923.pdf" target="_blank">Download</a></td></tr><tr><td>How to perform statistical analysis ? - Exploratory data analysis</td><td>Etienne Thévenot, Mélanie Pétéra, Jean-François Martin, Gildas Le Corguillé</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_2_ExploratoryDataAnalysis_150923.pdf" target="_blank">Download</a></td></tr><tr><td>How to perform statistical analysis ? -&nbsp;PCA analysis and&nbsp;(O)PLS(-DA) modelling</td><td>Etienne Thévenot</td><td><a href="/sites/workflow4metabolomics.org/files/files/w4m_HowToPerformStatisticalAnalysis_4_MultivariateModelling_150923.pdf" target="_blank">Download</a></td></tr><tr><td>Les banques d'annotation - Annotation</td><td>Yann Guitton, Franck Giacomoni</td><td><a href="/sites/workflow4metabolomics.org/files/files/W4M_Annotation_banques_annotation.pdf" target="_blank">Download</a></td></tr></tbody></table><p>&nbsp;</p>
                        </div>
                    </div>
                </div>
            </div>
            <hr />
            <div width="auto" align="center">
                <img src="W4M/workflow_all_HD_color_2.0.png" style="max-width:1500px;width:95%;" /><br/>
                <hr />
                <img src="W4M/workflow_community_HD_2.0.png" style="max-width:1000px;width:95%;" />
            </div>
        </div>


        <hr/>
        <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy</a> is an open, web-based platform for data intensive biomedical research.  The <a target="_blank" class="reference" href="http://wiki.galaxyproject.org/GalaxyTeam">Galaxy team</a> is a part of <a target="_blank" class="reference" href="http://www.bx.psu.edu">BX</a> at <a target="_blank" class="reference" href="http://www.psu.edu">Penn State</a>, and the <a target="_blank" class="reference" href="http://www.biology.emory.edu/">Biology</a> and <a target="_blank" class="reference" href="http://www.mathcs.emory.edu/">Mathematics and Computer Science</a> departments at <a target="_blank" class="reference" href="http://www.emory.edu/">Emory University</a>.  The <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy Project</a> is supported in part by <a target="_blank" class="reference" href="http://www.genome.gov">NHGRI</a>, <a target="_blank" class="reference" href="http://www.nsf.gov">NSF</a>, <a target="_blank" class="reference" href="http://www.huck.psu.edu">The Huck Institutes of the Life Sciences</a>, <a target="_blank" class="reference" href="http://www.ics.psu.edu">The Institute for CyberScience at Penn State</a>, and <a target="_blank" class="reference" href="http://www.emory.edu/">Emory University</a>.</p>

    </div>
</body>
</html>


<script>

    // modif : lecorguille
    // fonction toogle for diplay block <-> none
    function toogleDisplay(id)
    {
    if (document.getElementById(id).style.display == "block") {
        document.getElementById(id).style.display = "none";
    }
    else {
        document.getElementById(id).style.display = "block";
    }
    }
    // fin modif : lecorguille

</script>
